modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
I'm sorry sir, I'll stop issuing PRs without proper knowledge.. Should I close this PR?